src/Host/Client/Impl/Host/LocalBrokerClient.cs

```diff
@@ -24,14 +24,15 @@ internal sealed class LocalBrokerClient : BrokerClient {
         private const string InterpreterId = "local";
 
         private static readonly bool ShowConsole;
-        private static readonly LocalCredentialsDecorator Credentials = new LocalCredentialsDecorator();
+        private static readonly LocalCredentialsDecorator _credentials = new LocalCredentialsDecorator();
 
         private readonly string _rhostDirectory;
         private readonly string _rHome;
         private readonly BinaryAsyncLock _connectLock = new BinaryAsyncLock();
         private readonly ICoreServices _services;
 
         private Process _brokerProcess;
+        private int _disposed;
```

Comment:
I'll add `TryAdd` method

---
In reply to: [82298237](https://github.com/Microsoft/RTVS/pull/2447#discussion_r82298237) [](ancestors = 82298237,82297812)